R/estimate_KM.R

```diff
@@ -1,8 +1,7 @@
-#' @title Wrapper for Kaplan Meier analysis for an ADaM Basic Data Structure (BDS) for Time-to-Event analysis
+#' @title Wrapper for Kaplan Meier Time-to-Event analysis
```

Comment:
I would keep the references to CDISC standards as this is a big advantage that we can do this.

Reply:
I had the feeling that some core members wanted it to not be as present as it was (@bailliem?). But I might be wrong. I can leave it in for sure.